app/views/events/past_events.html.erb

```diff
@@ -0,0 +1,6 @@
+    <div class='past-events'>
+      <h1>Past events</h1>
+      <p>Hundreds of workshops in dozens of cities around the world have been
+      organized and taught by volunteers, based on our open source curricula.</p>
```

Comment:
Just a suggestion: if Bridge Foundry has an aggregation page that links to various curricula, we could throw in a link to that page here.

Reply:
Good idea @siruguri I did a check and it seems right now that page doesn't exist but we can keep an eye out for it in the future. Thanks for getting these PRs approved and have a stellar rest of your Sunday. :)